t/functional/serializers/test_registry.py

```diff
@@ -27,10 +28,12 @@ class Account(faust.Record):
     user: User
 
 
+async def get_json(record):
+    return await record.dumps(serializer='json')
+
 # Account1
 USER1 = User('A2', 'George', 'Costanza')
 ACCOUNT1 = Account(id='A1', active=True, user=USER1)
-ACCOUNT1_JSON = ACCOUNT1.dumps(serializer='json')
```

Comment:
I'd leave `ACCOUNT1_JSON` as is. You'll have a sometimes-awaitable object which you can await on as needed inside of the test case.

Reply:
Was my original idea but is not posible because of `dumps` is a `coroutine` and can be `awaited` only once. `pytest` does not "regenerate" the parametrization per test case, oor at least I could not find a way to do it. Do you have any ideas?

Reply:
Wrap it in a task? That means it runs immediately though, in case that has any impact on the processing. But you can await it more than once.

```
$ python -m asyncio
asyncio REPL 3.8.0 (default, Dec 10 2019, 09:39:53) 
[GCC 9.2.1 20191008] on linux
Use "await" directly instead of "asyncio.run()".
Type "help", "copyright", "credits" or "license" for more information.
>>> import asyncio
>>> async def foo():
...   print("foo ran")
...   return "bar"
... 
>>> f = asyncio.create_task(foo())
foo ran
>>> f
<Task finished name='Task-3' coro=<foo() done, defined at <console>:1> result='bar'>
>>> await f
'bar'
>>> await f
'bar'
```